symfony/config/routes/dev/web_profiler.yaml

```diff
@@ -0,0 +1,7 @@
+web_profiler_wdt:
+    resource: '@WebProfilerBundle/Resources/config/routing/wdt.xml'
+    prefix: /s/_wdt
+
+web_profiler_profiler:
+    resource: '@WebProfilerBundle/Resources/config/routing/profiler.xml'
+    prefix: /s/_profiler
```

Comment:
I had to modify these lines to include the `/s/` for it to work. I noticed the StarBRITE app didn't have to do this on its implementation, so I'm wondering if there's a different spot it is specified.